nominees/period-dignity-bristol.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Period Dignity Bristol",
+  "description": "A map app to connect people with free menstrual products",
+  "website": "https://www.periodfriendlybristol.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
```

Comment:
Related to my comment below, and for consistency with many other projects, we should choose one repo that is representative of the project, and include the license for that, even more so when all these 3 repos are released under the same license.

Reply:
Choose the backend repository to be representative of the project as documented in commit 5db5ad8.